frame.go

```diff
@@ -204,31 +204,29 @@ type StatusCode int
 // These codes were retrieved from:
 // https://www.iana.org/assignments/websocket/websocket.xhtml#close-code-number
 const (
-	StatusNormalClosure StatusCode = 1000 + iota
```

Comment:
This is 👍  since iota does make it harder to tell what number each status code corresponds.